connectors/es/sink.py

```diff
@@ -164,7 +164,6 @@ async def _bulk_api_call():
                 for op, data in item.items():
                     if "error" in data:
                         self._logger.error(f"operation {op} failed, {data['error']}")
-                        raise Exception(data["error"]["reason"])
```

Comment:
Should we adapt the log line above to something like this: `self._logger.error(f"Operation: {op} failed. Reason: {data['error']['reason']}. Full error: {data['error']}")`. Could be easier to spot the error reason faster instead of looking through the whole error object